.github/workflows/build.yml

```
@@ -362,18 +362,18 @@ jobs:
       run: make pythoninfo
     - name: Remount sources writable for tests
       # some tests write to srcdir, lack of pyc files slows down testing
-      run: sudo mount $CPYTHON_RO_SRCDIR -oremount,rw
+      run: sudo mount "$CPYTHON_RO_SRCDIR" -oremount,rw
     - name: Setup directory envs for out-of-tree builds
       run: |
-        echo "CPYTHON_BUILDDIR=$(realpath -m ${GITHUB_WORKSPACE}/../cpython-builddir)" >> $GITHUB_ENV
+        echo "CPYTHON_BUILDDIR=$(realpath -m "${GITHUB_WORKSPACE}"/../cpython-builddir)" >> "$GITHUB_ENV"
```

Do you need different quotes here for the nested strings?

```suggestion
        echo 'CPYTHON_BUILDDIR=$(realpath -m "${GITHUB_WORKSPACE}"/../cpython-builddir)' >> "$GITHUB_ENV"
```

Or just skip this one, since it's already inside a string?

```suggestion
        echo "CPYTHON_BUILDDIR=$(realpath -m ${GITHUB_WORKSPACE}/../cpython-builddir)" >> "$GITHUB_ENV"
```

---

It's this warning:
```
367 |       run: |
    |       ^~~~
.github/workflows/build.yml:367:7: shellcheck reported issue in this script: SC2086:info:1:83: Double quote to prevent globbing and word splitting [shellcheck]
```

Looking up the code: https://www.shellcheck.net/wiki/SC2086

This applies:

> Note that `$( )` starts a new context, and variables in it have to be quoted independently:
>
>     echo "This $variable is quoted $(but this $variable is not)"
>     echo "This $variable is quoted $(and now this "$variable" is too)"

---


> Or just skip this one, since it's already inside double quotes?

That would trigger this:

.github/workflows/build.yml:367:7: shellcheck reported issue in this script: SC2016:info:1:6: Expressions don't expand in single quotes, use double quotes for that [shellcheck]
    |
367 |       run: |
    |       ^~~~

---

Huh. It looks strange to me, but I'll trust you that this is the right thing to do!